docs/09.EXT-REFERENCE-ARG.md

```diff
@@ -335,6 +395,90 @@ jerryx_arg_native_pointer (void **dest,
 
 - [jerryx_arg_transform_this_and_args](#jerryx_arg_transform_this_and_args)
 
+## jerryx_arg_object_properties
+
+**Summary**
+
+Create a validation/transformation step (`jerryx_arg_t`) that expects to
+consume one `object` JS argument and call `jerryx_arg_transform_object_properties` inside
```

Comment:
`object` or `Object`? (`jerryx_arg_native_pointer` uses the capitalized form. I have no preference but for uniformity)